themes/odin-theme/static/css/dark.css

```diff
@@ -0,0 +1,5 @@
+/* dark.css: 👉 light-on-dark */
+:root {
+  --color: rgb(250, 250, 250);
```

Comment:
CSS variables not widely supported yet.

Reply:
https://caniuse.com/#feat=css-variables

Which browser do we have to support?